app/views/partials/messageInbox.scala.html

```diff
@@ -95,6 +95,10 @@
   padding:10px;
 }
 
+.default-text {
```

Comment:
can we not use the existing **.black-text** instead of adding another **.default-text** there?

Reply:
Yes we can ideed use that class. thanks